### ISSUE TITLE
Identify the set of packages we need to ship for UWP60

### ISSUE THREAD

**Issue by weshaggard** [dotnet/corefx#22670]
Ideally we would only ship the UWP platform package with the UWP6 release and avoid having to ship any of the other library nuget packages. We should do some analysis to see if there are any additional packages that have uwp specific assets in them that we need to ship. If we do have such packages we need to test to make sure they don't have netcoreapp specific assets as well as we need to ensure they still work on .NET Core 2.0.

cc @Petermarcu @danmosemsft

**Comment by joperezr** [comment]
Ok so after doing the investigation, I found packages that basically fall into one of three categories as follows:

- Packages which assets are not part of the UAP MP, and they have a `netstandard` build configuration that we package as UAP:
```
System.Configuration.ConfigurationManager
System.Security.Permissions
```

- Packages which assets are not part of the UAP MP, and they have a `uap` build configuration:
```
System.Data.SqlClient
System.Memory
System.Security.AccessControl
```

- Packages which assets are part of the RID-specific UAP MP only, but the ref asset is not in the UAP Identity MP:
```
System.Security.Cryptography.Cng
System.Security.Principal.Windows
```

**Comment by joperezr** [comment]
@weshaggard is the idea to test all 7 of them to make sure they work just fine with .NET Core 2.0? Or only the packages on the second and third bucket?

**Comment by joperezr** [comment]
cc: @ericstj as FYI.

**Comment by Petermarcu** [comment]
Question for me is why not just add the 2nd two categories to the MP?

**Comment by ericstj** [comment]
> and they have a netstandard build configuration that we package as UAP:

Shoudn't that be going away now that UAP is getting a netstandard2 mapping?  If so I suspect we wouldn't necessarily need to reship them (unless we needed to make fixes for UAP).

> System.Memory

Is this actually going stable?  If so then I think we should put it in the MP.  It'll have type-forwards to the core-assembly (I think) so we shouldn't really be shipping it in a package.

> Packages which assets are part of the RID-specific UAP MP only, but the ref asset is not in the UAP Identity MP

For these  (and SqlClient and AccessControl) I think we should just expose the ref in the MP as @Petermarcu suggests so long as they aren't PNS assemblies.

**Comment by joperezr** [comment]
> Question for me is why not just add the 2nd two categories to the MP?

We totally can do that. The reason why we didn't do it from the beginning is because the plan on constructing what went in the MP was:
 - Libraries that were there on the last shipped stable UAP MP
 - Full closure of netstandard.dll
 - assemblies that are required in order to make libraries from the above points api closure complete.

The four packages above (excluding System.Memory since I believe @ericstj is right and it is not stable yet) didn't meet any of the 3 above conditions which is why they are not currently in the MP but we can add them.

**Comment by joperezr** [comment]
> Shoudn't that be going away now that UAP is getting a netstandard2 mapping? If so I suspect we wouldn't necessarily need to reship them (unless we needed to make fixes for UAP).

Correct. That is why I wanted to put them in a separate bucket, as I suppose they will be a no-op once we have the mapping in place.

**Comment by Petermarcu** [comment]
Should we get the packages right right away and have people use "imports" to reference them if they need to?

**Comment by weshaggard** [comment]
Yes I'm fine moving those packages to the UAP platform package. It looks like a couple of them still have WACK violations that will need to be addressed.

Do we have a nuget issue to track the mapping? Or has it been added already?

**Comment by danmosemsft** [comment]
I know of no outstanding wack issues other than createfile and shgetknownfolderpath (on jeremy) and some that haven't got to rsmaster yet. @pjanotti has the details and can look at any you are referring to.

**Comment by weshaggard** [comment]
>I know of no outstanding wack issues other than createfile and shgetknownfolderpath (on jeremy) and some that haven't got to rsmaster yet. @pjanotti has the details and can look at any you are referring to.

This is the one I noticed while looking at the libraries in question https://github.com/dotnet/corefx/blob/release/uwp6.0/src/System.Security.Principal.Windows/src/PinvokeAnalyzerExceptionList.analyzerdata.uap

**Comment by danmosemsft** [comment]
They fixed that, not in master yet

**Comment by joperezr** [comment]
fixed by #22732
